docs/ndcube.rst

```diff
@@ -1,3 +1,6 @@
+.. doctest-skip-all::
+    all
```

Comment:
Should this still be skipping all the doctests??

Reply:
I am skipping all the doctests as we haven't rewritten the docs for the new APIs in NDCube 2 yet.